src/Admin/Settings.php

```diff
@@ -23,8 +23,29 @@ public static function caching_enabled() {
 		// get the cache_toggle setting
 		$option = function_exists( 'get_graphql_setting' ) ? \get_graphql_setting( 'cache_toggle', false, 'graphql_cache_section' ) : false;
 
+		$enabled = ( 'on' === $option );
+
+		$enabled = apply_filters( 'wpgraphql_cache_wordpress_cache_enabled', (bool) $enabled );
```

Comment:
Other filters in this plugin, start with 'graphql_cache_...'.

Reply:
ah ya, and the actions are a mix of `wpgraphql_cache` and `graphql_cache`

I'll change this, and make an issue to follow up and make things consistent across the board. 

I noticed the action `wpgraphql_cache_save_request` and changed these filters to follow that pattern, blindly assuming we were already consistent 😆

Reply:
Opened this issue to follow-up: https://github.com/wp-graphql/wp-graphql-smart-cache/issues/231

Reply:
@markkelnar ah yes, this is why I changed it to `wpgraphql_cache`: https://github.com/wp-graphql/wp-graphql-smart-cache/actions/runs/5555942329/jobs/10147786949?pr=230#step:7:20

Reply:
I reverted to `wpgraphql_cache` to be inline with the linter rules we have in place. 

I also updated #231 to remind us to consider the linter when we follow up to make all filter/action naming conventions the same